client/state/ui/language/schema.js

```diff
@@ -1,4 +1,4 @@
 /** @format */
 export const localeSlugSchema = {
-	anyOf: [ { type: 'string' }, { enum: [ false ] } ],
+	anyOf: [ { type: 'string' }, false ],
```

Comment:
I'd recommend simplifying the default state to `null` so we can simply do: ```{ type: ['string', 'null' ] };```

Reply:
Proposed the first option at https://github.com/Automattic/wp-calypso/pull/19319#discussion_r158358337.